examples/13-free_projection/fp_afqmc.ipynb

```diff
@@ -0,0 +1,1170 @@
+{
```

Comment:
Please clear the output of the cells. There are references to your local machine in the metadata too.